admin/controllers/restart.js

```diff
@@ -14,10 +14,7 @@ controller.getRestartOverview = async (req, res, next) => {
     return next(error)
   }
   let { hl } = req.query
-  if (hl) {
-    hl = JSON.parse(hl)
-    hl = typeof hl === 'string' ? JSON.parse(hl) : hl
-  }
+  hl = hl && hl.split(',')
```

Comment:
as discussed, can you please parse the UUIDs before trusting them in a query

Reply:
They get parsed by express as they are standard qs parameters.   From this point on they only get used by the display to match against properties to see if the green bar needs to be shown.

We could positively sanitise them against hex chars but is this necessary?

Reply:
as long as any inputs are sanitised, then i'm happy

Reply:
@GuyHarwood updated